### DISCUSSION TITLE
Disabling client-side navigation project-wide while still enabling client-side scripts

### DISCUSSION THREAD

**Discussion by mwcampbell** [remix-run/remix#4973] [General]
I'm starting a new web application project in which top-notch screen reader accessibility is a high priority. As such, I don't want to use client-side navigation anywhere in the app, since for screen reader users, actual browser navigation is still better. But I still want the other benefits of the `Link` component, such as building an `href` with both a path and search parameters. And when I hand off the project to a developer that's used to using `Link`, I don't want them to accidentally slip into using client-side navigation. So it would be helpful if there was a way to disable client-side navigation project-wide, without completely disabling JavaScript.

**Comment by sergiodxa** [comment]
You can add `reloadDocument` to both `Link` and `Form`, this will let you use those components while causing full page reloading instead of client-side navigations.

I would recommend you to wrap them in your own components Link and Form that always set them so you don't forget.
